api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -92,6 +92,9 @@ message UserMetadata {
 
   // RequiredPrivateKeyPolicy is the private key policy enforced for this login.
   string RequiredPrivateKeyPolicy = 9 [(gogoproto.jsontag) = "required_private_key_policy,omitempty"];
+
+  // Bot indicates this user is a Machine ID bot user.
+  bool Bot = 10 [(gogoproto.jsontag) = "bot,omitempty"];
```

Comment:
After only wasting a day debugging why bot events are always being reported as `human` rather than `unspecified` for outdated nodes... this needs to be changed to an enum as used in the prehog protos to account for outdated clients.

Reply:
Alright, this appears to be fixed now, verified on a Cloud tenant with both a vanilla v13 node which produces `user_kind: ""` aka unspecified and a node running this branch which correctly indicates bot/human.